Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM	rust:1 AS builder
```

Comment:
This will cause pain when Rust 1.32 is released, as the 1.31 image will no longer get updates of any kind.

One usually pins the branch where backwards compatibility is guaranteed in the Dockerfile. I don't do Rust myself, but I suspect that 1.x will be fully backwards compatible (modulo bugs).

Reply:
Ah, that's a good point. The release itself will still be supported (with patch releases in case there are major bugs), but the Docker images won't be re-built.

In that case I agree that sticking with `1` is probably fine.